microsoft-edge/webview2/how-to/runtime-channel.md

```diff
@@ -0,0 +1,188 @@
+
+### WPF
+
+WPF uses an approach similar to the Win32/C++ approach described above.
+
+See [CoreWebView2CreationProperties.BrowserExecutableFolder Property](https://docs.microsoft.com/dotnet/api/microsoft.web.webview2.wpf.corewebview2creationproperties.browserexecutablefolder?view=webview2-dotnet-1.0.864.35#Microsoft_Web_WebView2_Wpf_CoreWebView2CreationProperties_BrowserExecutableFolder).
+
+
+<!--========================================================================-->
+## Using a group policy
+
+To make your application use the Canary or Dev channel of Microsoft Edge by using a group policy, you copy ADMX and ADML files to PolicyDefinitions, as follows.
+
+1.  Get the policy files.  To do this, download policy files from [Download and deploy Microsoft Edge for business](https://www.microsoft.com/edge/business/download).  Or, find generated policy files in `out\debug_x64\gen\chrome\app\policy\windows\admx` after running `autoninja edge_webview2_policy_templates`.
+
+1.  Copy the ADMX file into a Policy Definitions template folder, such as `C:\Windows\PolicyDefinitions`.
+
+1.  Copy the ADML file into a matching locale folder within the `Policy Definitions` folder, such as a `C:\Windows\PolicyDefinitions\en-us` folder.
+
+1.  Configure policies using the **Local Group Policy Editor** or by using command-line tools (see below).<!--below, where?-->
+
+    The registry location is: `{HKLM/HKCU}\Software\Policies\Microsoft\Edge\WebView2\{override name}`
+
+For more information, see [Configure Microsoft Edge policy settings](https://docs.microsoft.com/DeployEdge/configure-microsoft-edge).
+
+<!-- link to Enterprise docs re: Group Policy? -->
+
+
+<!--========================================================================-->
+## Using a registry override
+
+To make your application use the Canary or Dev channel of Microsoft Edge by using a registry override:
+
+1.  Open a Powershell command prompt.
+
+1.  Run the following command:
+
+    `REG ADD HKLM\Software\Policies\Microsoft\Edge\WebView2\BrowserExecutableFolder /v * /t REG_SZ /d "C:\Users\myname\AppData\Local\Microsoft\Edge SxS\Application\88.0.680.0"`
+
+1.  Replace `C:\Users\myname\AppData\Local\Microsoft\Edge SxS\Application\88.0.680.0` by the path to the desired Runtime channel.<!--should this instead be "Microsoft Edge channel"?-->
+
+
+### Resuming using the default, WebView2 Evergreen Runtime
+
+To undo the above setting, run the following command:
+
+`REG DELETE HKLM\Software\Policies\Microsoft\Edge\WebView2\BrowserExecutableFolder /f`
+
+
+<!--========================================================================-->
+## Using an environment variable
+
+If you use the environment-variable approach, you cannot specify a specific runtime path, but rather, you reverse the search order for the Runtime channel.<!-- "runtime channel" terminology ok?  "WebView2 Runtime or Microsoft Edge channel"?-->
+
+The default channel-search order when using the above, registry-override approach is:
```

Comment:
This is just the general search order that is applied if browser executable folder is not specified by any of the methods you describe. If there is no browser executable folder provided, the next thing we check is if there are any overrides for release channel preference, which would reverse the search order. You can set release channel preference with a registry override, env variable, or group policy